package/collection2/package.js

```diff
@@ -18,7 +17,7 @@ Npm.depends({
 Package.onUse(function(api) {
```

Comment:
Could we use `api.versionsFrom` here, instead of specific versions of core packages?

Reply:
Done, set it to `1.12.1` for now. We can discuss if we want to go lower.